python-backend/app/api/mines/variances/resources/variance.py

```diff
@@ -107,85 +76,44 @@ def get(self, mine_guid, variance_id):
         return variance
```

Comment:
This is related to the URL design discussion, the mine_guid is in the URL but isn't used to respected or used. I may comment on this after that discussion.